src/helpers.js

```diff
@@ -6,17 +6,18 @@ var Helpers = {
    * @return {Object}
    */
   extend: function(destination, source) {
-    var sources = Array.prototype.slice.call(arguments, 1) || [];
+    if (source) {
```

Comment:
seems that the previous version was fine too, i mean, if there's no `source` parameter, `sources = []`, so nothing to loop on.